docs/website/docs/general-usage/credentials/config_specs.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Configuration Specs
+description: Overview configuration specs and how to create custom specs
+keywords: [credentials, secrets.toml, secrets, config, configuration, environment
+      variables, specs]
+---
+
+# Configuration Specs
+
```

Comment:
Also here, s in docs/website/docs/general-usage/credentials/configuration.md, it's best to welcome the reader and begin the narrative with a brief summary or description of what the concept is. What are configuration specs?